packages/core/upload/server/services/image-manipulation.js

```diff
@@ -172,7 +185,8 @@ const isSupportedImage = async file => {
 };
 
 module.exports = () => ({
-  isSupportedImage,
+  isOptimizableImage,
```

Comment:
I have seen users using this: to not break their use-cases should we keep `isSupportedImage()` and render a deprecation warning until v5?